src/main/java/com/ericsson/ei/controller/SubscriptionControllerImpl.java

```diff
@@ -81,6 +81,7 @@
             try {
                 LOG.debug("Subscription creation has been started: " + subscriptionName);
                 subscriptionValidator.validateSubscription(subscription);
+				subscriptionValidator.validateWithSchema(subscription);
```

Comment:
Please revert changes in this file. there should not be any tabs.

Reply:
Done